pkg/commands/resolver.go

```diff
@@ -172,6 +176,9 @@ func makePublisher(po *options.PublishOptions) (publish.Interface, error) {
 	// to either a docker daemon or a container image registry.
 	innerPublisher, err := func() (publish.Interface, error) {
 		repoName := po.DockerRepo
+		if po.DockerRepoFromConfig != "" {
+			repoName = po.DockerRepoFromConfig
+		}
```

Comment:
If this is the only thing we use the two fields for, I'm not sure we need it. Can we just set `po.DockerRepo` to the value we get from the env or config?

Reply:
thank you fixed right now